RabbitMQ.pm

```diff
@@ -109,6 +115,8 @@ Returns the maximum allowed channel number.
 
 =head2 exchange_declare($channel, $exchange, $options, $arguments)
 
+Declare an AMQP exchange on the RabbitMQ server.
```

Comment:
Maybe add "unless it already exists"? Maybe add stuff about how this goes boom if the name exists with different parameters?